src/components/httproute/HTTPRouteCreatePage.tsx

```
@@ -0,0 +1,733 @@
+                      <Th width={30}>{t('Backend references')}</Th>
+                      <Th width={10}>{t('Actions')}</Th>
+                    </Tr>
+                  </Thead>
+                  <Tbody>
+                    {rules.map((rule, index) => (
+                      <Tr key={rule.id || index}>
+                        <Td dataLabel={t('Rule ID')}>
+                          <strong>{rule.id}</strong>
+                        </Td>
+                        <Td dataLabel={t('Matches')}>
+                          <span style={{ color: rule.matches?.length > 0 ? 'inherit' : 'var(--pf-v6-global--Color--200)' }}>
```

ESLint

[error] Replace `·style={{·color:·rule.matches?.length·>·0·?·'inherit'·:·'var(--pf-v6-global--Color--200)'·}}` with `⏎····························style={{⏎······························color:⏎································rule.matches?.length·>·0⏎··································?·'inherit'⏎··································:·'var(--pf-v6-global--Color--200)',⏎····························}}⏎··························`

(prettier/prettier)